website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -295,17 +295,6 @@ spec:
 Note that when using the `Custom` AMIFamily you will need to specify fields **both** in `spec.kubelet` and `spec.userData`.
 {{% /alert %}}
 
-{{% alert title="Warning" color="warning" %}}
```

Comment:
Can we remove from all version of the docs?

Reply:
Never mind, realized this is a new feature

Reply:
This change is only on the latest version so I'm not sure if it makes sense to update all versions of the docs